src/services/request.services.js

```diff
@@ -24,6 +24,87 @@ class TripRequestService {
     const updatedRequest = Queries.updateTripRequestStatus(db.requesttrip, status, requestId);
     return updatedRequest;
   }
+
+  /** Function to update the status to approved or reject
+   *
+   * @param {string} tripId the id of the selected trip
+   * @param {Object} body it gets the data from req.body
+   * @returns {object} updated data
+   */
+  static async updateTrip(tripId, body) {
+    const data = {
+      reason: body.reason,
+      originId: body.From,
+      destinationId: body.To,
+      accomodationId: body.accomodationId,
+      departureDate: body.departureDate,
+      returnDate: '' || body.returnDate,
+      tripType: body.type
+    };
+    const updatedRequest = Queries.updateTrip(db.trips, tripId, data);
+    return updatedRequest;
+  }
+
+  /**
+   * This method will provide a service of updating multi city
+   * a trip data in database
+   * @param {Object} body data posted by user
+   * @param { integer } tripId trip id as integer
+   * @param {Object} tripType type of trip
+   * @returns { Object } response data
+   */
+  static async updateMultiCityTripRequest(body, tripId) {
+    const data = {
```

Comment:
the same here  `updateMultiCityTripRequest`  and `updateTrip` methods are doing the same thing, so you can find a way of using a single method.